lib/Service/Appointments/MailService.php

```diff
@@ -124,9 +124,9 @@ public function sendConfirmationEmail(Booking $booking, AppointmentConfig $confi
 		$bookingUrl = $this->urlGenerator->linkToRouteAbsolute('calendar.booking.confirmBooking', ['token' => $booking->getToken()]);
 		$template->addBodyButton($this->l10n->t('Confirm'), $bookingUrl);
 
-		$template->addBodyListItem($user->getDisplayName(), 'Appointment with:');
+		$template->addBodyListItem($user->getDisplayName(), $this->l10n->t('Appointment with:'));
 		if (!empty($config->getDescription())) {
-			$template->addBodyListItem($config->getDescription(), 'Description:');
+			$template->addBodyListItem($config->getDescription(), $l10n->t('Description:'));
```

Comment:
```suggestion
			$template->addBodyListItem($config->getDescription(), $this->l10n->t('Description:'));
```

and for all following usages of `$l10n`